dotmanager/constants.py

```diff
@@ -43,8 +43,14 @@
 import os
 import sys
 from dotmanager.types import Path
-from dotmanager.utils import normpath
+from dotmanager.utils import normpath, find_files
 
+# Search paths for config files
+CONFIG_SEARCH_PATHS = [
+    os.path.join(os.path.dirname(os.path.dirname(sys.modules[__name__].__file__), "data")),
+    "/etc/dotmanager",
+    os.path.join(os.getenv('XDG_CONFIG_HOME', os.path.expanduser('~/.config')), "dotmanager")
```

Comment:
When dealing with environment vars or ~ you should use the functions `utils.get_user_env_var()` and `utils.expanduser()`. The problem is that dotmanager will start itself with sudo when for example links need to be created in `/etc`. In this case environment vars would change and `os.path.expanduser('~/.config')` would evaluate to `/root/.config`